pkg/deploy/api/test/ok.go

```diff
@@ -24,7 +24,7 @@ func OkDeploymentConfig(version int64) *deployapi.DeploymentConfig {
 		ObjectMeta: kapi.ObjectMeta{
 			Name:      "config",
 			Namespace: kapi.NamespaceDefault,
-			SelfLink:  "/oapi/v1/namespaces/default/deploymentconfig/config",
+			//SelfLink:  "/oapi/v1/namespaces/default/deploymentconfig/config",
```

Comment:
maybe, but it is not needed here... i should probably remove the commented selfLink